### DISCUSSION TITLE
Exclude some global CSS files when CSS Modules is default

### DISCUSSION THREAD

**Discussion by thien-do** [parcel-bundler/parcel#8508] [Q&A]
In my project I have 2 CSS files that should be global (e.g. to target a 3rd-party library, like monaco-editor) and the rest 98 files should be modules. Therefore, I [enabled CSS modules globally](https://parceljs.org/languages/css/#enabling-css-modules-globally) but then I don't know how to escape those 2 files. I tried `name.global.css` but it does not work.

In other words:
- in package.json, cssModules is `true`
- foo.css should be modules -> correct
- foo.global.css should be global -> how?

**Comment by devongovett** [comment]
If it's for a third party library in node_modules, then this should already be the case. CSS modules are never enabled in node_modules, unless the filename ends with `.module.css`.

**Reply by thien-do** [reply]
Hi @devongovett unfortunately it is not inside node_modules. I'm sorry for my lack of clarification. The CSS I want is my CSS written to target a 3rd-party library.

Imagine there is a library that inserts a widget in a website. This widget is under a `<div class="foo" />`. Therefore, I'd like to customise it by doing:

```css
.foo { }
```

For 1 selector, I can use `:global(.foo)`. However, imagine that this is a complex widget (like an editor) that has dozens of elements that I want to target via class names. This is why I'm looking for a way to escape CSS Modules for just a few files.
